functions/go/inflate-helm-chart/third_party/sigs.k8s.io/kustomize/api/builtins/HelmChartInflationGenerator.go

```diff
@@ -77,31 +61,27 @@ func (p *HelmChartInflationGeneratorPlugin) establishTmpDir() (err error) {
 	return err
 }
 
-func (p *HelmChartInflationGeneratorPlugin) validateArgs() (err error) {
+func (p *Plugin) ValidateArgs() (err error) {
 	if p.Name == "" {
 		return fmt.Errorf("chart name cannot be empty")
 	}
 
-	// ChartHome might be consulted by the plugin (to read
-	// values files below it), so it must be located under
-	// the loader root (unless root restrictions are
-	// disabled, in which case this can be an absolute path).
+	// ChartHome might be written to by the function in a container,
+	// so it must be under the `/tmp` directory
 	if p.ChartHome == "" {
-		p.ChartHome = "charts"
+		p.ChartHome = "tmp/charts"
```

Comment:
Should it be `/tmp/charts`?

Reply:
It needs to be relative to run well as an exec function. If a user runs it with kustomize as an exec, it will use the local `./tmp/charts/` directory.

When run in a container, the `absPath` method is able to convert this to `/tmp/charts`.